### PR TITLE
Feature/icloud backup

### PR DESCRIPTION
**Pull request by Kevincamp**
## Purpose
CloudKit Services with Background Processing Task

## Bugfix

- If this is a bugfix branch, fill this out instead of the above "Purpose" section.
- Describe what issue is being fixed

## Related tasks

- #1290 Add Backup In iCloud

### CODE REVIEW

**Review by GianniCarlo — CHANGES_REQUESTED** [review]
I have more feedback regarding the cloudkit service, but I'll get into that after the PR is updated with the current feedback. Also, tests are failing because the main target has minimum deployment of iOS 15.6, but the rest that are related to iOS are set to 14.0. Let's set them to 15.0

File: BookPlayer/BookPlayer.entitlements [hunk 1]
```diff
@@ -2,30 +2,33 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>aps-environment</key>
```
[the remark this answers is not in the document]
Comment (Kevincamp) [reply BookPlayer/BookPlayer.entitlements:5]: these changes setted once that I configure the provisional profile, I will roll back if needed

**Comment by GianniCarlo** [comment]
we'll be using local backups as an alternative, introduced in #1384
